pushover.sh

```diff
@@ -31,6 +32,7 @@ usage() {
     echo " -u <url>"
     echo " -U <USER> (required if not in config file)"
     echo " -a <url_title>"
+	echo " -P <http://proxy:port/>"
```

Comment:
Actually, why use a full URI? Curl's own proxy definition is simply <host>:<port>. I would prefer to keep with that syntax, as the http:// bit doesn't add any information (and is stripped out)